testing/tests/DevExpress.ui.widgets/toolbar.tests.js

```diff
@@ -242,105 +238,6 @@ QUnit.module("toolbar with menu", {
     }
 });
 
-QUnit.test("render menu items", function(assert) {
```

Comment:
Breaking change?

Reply:
I think it doesn't BC by the same reason https://github.com/DevExpress/DevExtreme/pull/7497#discussion_r274313416